processing/src/main/java/io/druid/query/groupby/epinephelinae/ConcurrentGrouper.java

```diff
@@ -94,6 +96,8 @@ public ConcurrentGrouper(
           )
       );
     }
+
+    this.entryComparator = keySerdeFactory.factorize().entryComparator();
```

Comment:
@gianm ah, in that case, we will have to have one comparator on KeySerdeFactory and other on KeySerde . ok let us just move entryComparator aka objectComparator to KeySerdeFactory and not have it in KeySerde (we can add it there if ever needed).
does that make sense?